.github/workflows/ci.yml

```diff
@@ -144,6 +146,7 @@ jobs:
           DOCKER_PASSWORD: ${{ secrets.DOCKER_PASSWORD }}
 
   docs:
+    needs: test
```

Comment:
Very important: docs need to run asap, since they are very slow, so we want to kickstart them asap. If one of the jobs fail, docs will be killed anyways.